pkg/apiserver/slowquery/queries.go

```diff
@@ -109,11 +109,19 @@ func QuerySlowLogList(req *GetListRequest, sysSchema *utils.SysSchema, db *gorm.
 func QuerySlowLogDetail(req *GetDetailRequest, db *gorm.DB) (*Model, error) {
 	var result Model
 	err := db.
-		Select("*, (UNIX_TIMESTAMP(Time) + 0E0) AS timestamp").
+		Select("*, (UNIX_TIMESTAMP(Time) + 0E0) AS timestamp, tidb_decode_binary_plan(Binary_plan) AS Binary_plan_text").
 		Where("Digest = ?", req.Digest).
 		Where("Time = FROM_UNIXTIME(?)", req.Timestamp).
 		Where("Conn_id = ?", req.ConnectID).
 		First(&result).Error
+	if err != nil {
+		err = db.
+			Select("*, (UNIX_TIMESTAMP(Time) + 0E0) AS timestamp").
+			Where("Digest = ?", req.Digest).
+			Where("Time = FROM_UNIXTIME(?)", req.Timestamp).
+			Where("Conn_id = ?", req.ConnectID).
+			First(&result).Error
+	}
```

Comment:
Is it to eliminate any errors generated by `tidb_decode_binary_plan`? What errors does it generate?

Reply:
This method may not exist in the old tidb version, so it's for compatibility.